docs/pages/setup/deployments/gcp.mdx

```diff
@@ -192,29 +189,41 @@ teleport:
     audit_events_uri: 'firestore://events?projectID=example_Teleport-Project-Name&credentialsPath=/var/lib/teleport/gcs_creds.json'
     audit_sessions_uri: 'gs://teleport-session-storage-2?projectID=example_Teleport-Project-Name&credentialsPath=/var/lib/teleport/gcs_creds.json'
 auth_service:
-  enabled: yes
-  auth_service:
-    tokens:
-    - "proxy,node:EXAMPLE-CLUSTER-JOIN-TOKEN"
+  enabled: true
+  tokens:
+    - "proxy:EXAMPLE-PROXY-JOIN-TOKEN"
```

Comment:
Fixed the token to match in 7e607f2

Reply:
I agree philosophically with the idea of using dynamic tokens instead of static tokens, but honestly I'm not trying to boil the ocean in terms of fixing the doc here - I just wanted to take a broken config that wouldn't work for anyone trying it and turn it into something that will at least work out of the box, even if it's not the most secure method. Happy to be overruled if you don't think it's acceptable.

Reply:
👍 Eventually this guide will need a more comprehensive set of edits, but for now, thanks for making it more usable.